src/renderer/containers/TemplateEditorModal/index.tsx

```diff
@@ -54,6 +54,9 @@ const styles = require("./styles.pcss");
 
 const { Search } = Input;
 
+const ADD_ANNOTATION_DESCRIPTION = `Certain annotations trigger automated processes for new uploads.
+Optical Control ID and Is Optical Control are used to submit images for automatic alignment.
+Well is used to automatically add metadata from Labkey.`
```

Comment:
I'm now wondering if "Well" should just be excluded altogether since it is included by default and we really don't want them adding "Well" to templates.

Reply:
How do you know "Well" is a default? It appears in the list of annotations to add to a new template, and I see templates without Well (e.g., AICS CofA Images). Only Notes and Plate Barcode are included in all of the current templates

Reply:
Well isn't shown until a Plate Barcode is selected so by that reasoning it is always included since Plate Barcode is always included.

Reply:
Ohhh, great